### PR TITLE
Fix for upside-down RGB images

### PR DESCRIPTION
**Pull request by ejeschke**
- fix for issue #713

### CODE REVIEW

**Comment by ejeschke** [comment]
@pllim, the earlier fix for the upside down Ginga banner page did not fix this issue, but I think this PR should fix both that and issue #713.  Can you test?  Be sure to remove any `.thumbs` directory that ginga creates in your test data directory or `$HOME/.ginga`.

**Comment by pllim** [comment]
With this branch, table icon seems to be okay but Ginga banner is upside-down again.

**Comment by ejeschke** [comment]
@pllim, do you have `opencv` installed or not?

**Comment by ejeschke** [comment]
If so, could you tell me the version? Tnx

**Comment by pllim** [comment]
With your new commit, both table icon and Ginga banner look fine now. Thanks!

FWIW:
```
$ conda list opencv
# packages in environment at .../anaconda/envs/py37:
#
# Name                    Version                   Build  Channel
libopencv                 3.4.2                hb342d67_1  
opencv                    3.4.2            py37h6fd60c2_1  
py-opencv                 3.4.2            py37hb342d67_1  
```

**Comment by pllim** [comment]
Just need to fix that PEP 8 😸 
```
ginga/util/io_rgb.py:52:1: E302 expected 2 blank lines, found 1
```

File: ginga/util/io_rgb.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
 #have_pil = False
 #have_cms = False
 #have_exif = False
-
+#have_opencv = False
```
Comment (pllim) [inline ginga/util/io_rgb.py]: I think this should fix the PEP 8 failure?
```suggestion
#have_opencv = False

```

Comment (ejeschke) [reply ginga/util/io_rgb.py]: Just pushed a fix for that, let's see if Travis is happy...

**Comment by ejeschke** [comment]
I'm going to try to backport this to the v2.x branch as well.

**Comment by pllim** [comment]
Please add a milestone to this PR to the version you want it to be in, for future reference. Thanks!

**Pull request merged**